src/test/java/seedu/address/logic/commands/ExportCommandTest.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.ExportCommand.MESSAGE_SUCCESS;
+import static seedu.address.testutil.TypicalFileTypes.FILE_TYPE_CSV;
+import static seedu.address.testutil.TypicalFileTypes.FILE_TYPE_VCF;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code ExportCommand}.
+ */
+public class ExportCommandTest {
+    private Model model = new ModelManager();
+    private Model expectedModel = new ModelManager();
+
+    @Test
+    public void execute_export_success() {
+        CommandResult expectedCommandResult = new CommandResult(String.format(MESSAGE_SUCCESS, FILE_TYPE_CSV),
+            false, false);
+        assertCommandSuccess(new ExportCommand(FILE_TYPE_CSV), model, expectedCommandResult, expectedModel);
+    }
+
+    @Test
+    public void toCsvStringMethod() {
+        ExportCommand exportCommand = new ExportCommand(FILE_TYPE_CSV);
+
+        String tag = "\"friends\"";
+        String note = "\"High profile client, Likes dumplings\"";
+
+        String expected = ALICE.getName() + "," + ALICE.getPhone() + "," + ALICE.getEmail() + ",\""
+            + ALICE.getAddress() + "\"" + "," + tag + "," + note;
```

Comment:
Unfortunately to be consistent with how the rest of the test cases are written, I suggest that the tag and note be some constant pulled from `TypicalPersons`:

1. Add valid notes to `CommandTestUtil`, i.e. `VALID_NOTE_HIGH_PROFILE`, etc.
2. Create a new `TypicalPerson` called `CHARLIE` with the valid notes and valid tags
3. Refactor this part of the code to use `CHARLIE` instead